warehouse/query-core/src/main/java/datawave/query/tables/RemoteEventQueryLogic.java

```diff
@@ -69,16 +68,19 @@ public RemoteEventQueryLogic clone() {
 
     @Override
     public Set<String> getOptionalQueryParameters() {
+        // @TODO Get an instanceof the underlying query logic?
```

Comment:
@ivakegg is this still a TODO?

Reply:
This was already removed in the main line.  Perhaps a bad merge somewhere along the line.  I will remove it here.

Reply:
done

Reply:
cool, thanks.  i did many, many, MANY merges so no surprise to me if i merged something in an undesirable way.